### ISSUE TITLE
Object-detection validation loss curve increasing

### ISSUE THREAD

**Issue by nanditam1** [xingyizhou/CenterNet#163]
I am training ctdet-dla-34 from the ctdet_coco_dla_2x.pth pretrained weights for 1 class object detection, for a large dataset in coco format. However the validation loss seems to increase as below:
![training_full_issue](https://user-images.githubusercontent.com/38249335/60084180-abfc5a80-9754-11e9-9847-6189c4481dae.png)

Any clues on why this is happening?

**Comment by xingyizhou** [comment]
See https://github.com/xingyizhou/CenterNet/issues/148. Loss is not aligned well with AP. We also observed the loss starts increasing in early epochs, but usually the AP will be better when trained longer. It is recommended to try plotting your AP.
